includes/kp-functions.php

```diff
@@ -487,3 +487,71 @@ function ( $feature ) {
 
 	return $unavailable_features;
 }
+
+/**
+ * Set the payment method title for a Klarna order.
+ *
+ * @param WC_Order $order WooCommerce order.
+ * @param array    $klarna_place_order_response The Klarna place order response.
+ * @return void
+ */
+function kp_set_payment_method_title( $order, $klarna_place_order_response ) {
+	$klarna_order_id = $klarna_place_order_response['order_id'];
+	$response        = KP_WC()->api->get_klarna_om_order( $order->get_billing_country(), $klarna_order_id );
```

Comment:
Question: Do we really need to make another request here? Can't we get this information from the previous requests or pass it down from further up? Adding another request during the order confirmation will make the time to show the thankyou page longer and should be avoided if possible.